packages/bp/src/core/events/queue/memory-queue.test.ts

```diff
@@ -93,7 +93,7 @@ describe('Lite Queues', () => {
       await Promise.delay(5)
     }
 
-    await Promise.delay(5)
+    await Promise.delay(20)
```

Comment:
PP: Jest has a fake timer mechanism (https://jestjs.io/docs/timer-mocks) that I think we should prioritize instead of using "real" Promise.delay

Reply:
Ah, I didn't knew that ! We use delays in a lot of tests, what do you think of keeping this PR this way, then use a different PR to update the multiple places where we use those timers? It would take some time to make sure it works correctly and change the various places we use it

Reply:
Totally agree!